### PR TITLE
Add new GitHub profile readme

### PR DESCRIPTION
**Pull request by olafurpg**
This is a new GitHub feature https://github.blog/changelog/2021-09-14-readmes-for-organization-profiles/

### CODE REVIEW

**Review by tgodzik — APPROVED** [review]
:rocket:

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Code of conduct
+
+The Scalameta organization is committed to provide a safe and welcoming
+environment for all people to learn, teach and contribute. We follow the
+[Scala code of conduct](https://www.scala-lang.org/conduct/). Don't hesitate to
+reach out to our organization admins to raise a problem or ask a question.
+
+| Organization admin                                  | Contact                |
+| --------------------------------------------------- | ---------------------- |
+| 🇺🇸 [Chris Kipp](https://github.com/ckipp01)         | ckipp@pm.me            |
```
Comment (ckipp01) [inline profile/README.md]: 😬 Can you swap this?
```suggestion
| 🇺🇸 [Chris Kipp](https://github.com/ckipp01)         | open-source@chris-kipp.io            |
```
If not merge and I can update as well.

**Review by gabro — APPROVED** [review]
🥇

**Review by ckipp01 — APPROVED** [review]
💯

File: profile/README.md (hunk 1 above)
Comment (olafurpg) [reply profile/README.md]: Done.

**Pull request merged**